packages/server/lib/modes/run.ts

```diff
@@ -742,6 +743,7 @@ async function runSpecs (options: { config: Cfg, browser: Browser, sys: any, hea
     specPattern,
     system: _.pick(sys, 'osName', 'osVersion'),
     tag,
+    autoCancelAfterFailures,
```

Comment:
The https://docs.cypress.io/api/plugins/before-run-api docs need updated to include information on `autoCancelAfterFailures`

Reply:
The BeforeRun types also need updated: https://github.com/cypress-io/cypress/blob/alejandro/feat/add-autoCancelAfterFailures-flag/cli/types/cypress.d.ts#L5768